circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 6.9.4
```

Comment:
slack is handled by circle but not in this config
also all servers with node will be using lts so didnt see a reason to test btu could be added in future